kafka-rest/src/main/java/io/confluent/kafkarest/controllers/ClusterConfigManagerImpl.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.kafkarest.controllers;
+
+import io.confluent.kafkarest.entities.ClusterConfig;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import javax.inject.Inject;
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.common.config.ConfigResource;
+
+final class ClusterConfigManagerImpl
```

Comment:
Reason for not this class to be `public` ?

Reply:
All `Impl` classes are package-private :)
Classes should have minimal visibility. In the case of these `Impl` classes, they are only referenced at the `Module` class in the same package.